src/SimpleAmqpClient/Channel.h

```diff
@@ -29,6 +29,7 @@
  * ***** END LICENSE BLOCK *****
  */
 
+#include <amqp.h>
```

Comment:
Don't add rabbitmq-c includes to public SimpleAmqpClient headers (it makes the interface leaky).